FirebasePerformance.podspec

```diff
@@ -32,6 +32,7 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
     'FirebaseCore/Sources/Private/*.h',
     'FirebaseInstallations/Source/Library/Private/*.h',
     'FirebaseRemoteConfig/Sources/Private/*.h',
+    'Interop/Crashlytics/Public/*.h',
```

Comment:
Fine for hackweek, but the Interop directory is deprecated and new ones should be created like FirebaseAppCheck/Sources/Interop

Reply:
Ooh cool thanks for that tip